actors/user/src/main/java/org/sunbird/user/actors/UserStatusActor.java

```diff
@@ -75,10 +58,11 @@ private void unblockUser(Request actorMessage) {
 
   private void updateUserStatus(Request request, boolean isBlocked) {
     String operation = request.getOperation();
-    
-    Util.getUserProfileConfig(systemSettingActorRef);
+
+    Util.getUserProfileConfig(getSystemSettingActorRef());
```

Comment:
Why do we need to call getUserProfileConfig?

Reply:
Having this method in Util also doesn't seem right anymore.

Reply:
In this case - the call is not required - i am removing the call - as block/unblock is not a profile field - this is required for userprofile fields, i.e. private/public fields configuration only. 

Regardless of that, what is the correct place to move this method to?
Where do u suggest to move this?